README.md

```diff
@@ -237,46 +253,29 @@ public function validateResponse (http:Caller outboundEp, http:Response res) {
 
 Similarly the API developer can add interceptors globally at the API level as well
 ```
-apis:
-    /petstore/v1:
-        title: Swagger Petstore New
-        version: 1.0.0
-        production_endpoint:
-            type: 'http'
-            urls:
-                - 'https://petstore.swagger.io/v2'
-        sandbox_endpoint:
-            type: 'http'
-            urls:
-                - 'https://sand.petstore.ballerina.io/'
-        request_interceptor: validateRequest
-        response_interceptor: validateResponse
-        security: 'oauth'
+openapi: 3.0.0
+  version: 1.0.0
+  title: Swagger Petstore New
+  termsOfService: http://swagger.io/terms/
+x-mgw-basePath: /petstore/v1
+x-mgw-request-interceptor: validateRequest
+x-mgw-response-interceptor: validateResponse
+x-mgw-production-endpoints:
+  urls:
+  - https://petstore.swagger.io/v2
 ```
 
-#### 3. Add resource level throttling policies
-API developer can specify the rate limiting policies for each resource. These policies should be defined in the policies.yaml file in the project directory
-By default set of policies are available, but user can add more policies to the file and later refer them by name in the definition.yaml
-
+Sample open API definition for interceptors can be found [here](samples/interceptors_sample.yaml).
```

Comment:
IMO, It would be better to mention the structure of the interceptor ballerina function.

Reply:
There are samples of interceptor functions in the README file